fastlane/Fastfile

```diff
@@ -6,8 +6,7 @@ lane :setup do
     sh 'git clone https://github.com/artsy/emission.git'
     Dir.chdir('emission') do
       sh 'git checkout skip'
-      sh 'pod repo update'
-      sh 'npm install'
+      sh '. ~/.nvm/nvm.sh && nvm use && npm install yarn --global && yarn install'
```

Comment:
On travis CI - yarn would take longer than 10m to install on a the Mac CIs, and would cancel the builds (which is why I had to use npm) though I was using the official route rather than npm. So let's see where that goes.